src/lib/utils.ts

```diff
@@ -4,3 +4,11 @@ import { twMerge } from 'tailwind-merge';
 export function cn(...inputs: ClassValue[]) {
     return twMerge(clsx(inputs));
 }
+
+export function cloneValue<T>(value: T): T {
+    if (typeof value !== 'object' || value === null) {
+        return value;
+    }
+
+    return JSON.parse(JSON.stringify(value));
```

Comment:
this is only used to send values via the context bridge since it would fail otherwise because of some electron errors and will not be used anywhere else

Reply:
yes you can add that

Reply:
this fits the context